jetbrains-core/src/software/aws/toolkits/jetbrains/utils/NotificationUtils.kt

```diff
@@ -17,26 +18,26 @@ import software.aws.toolkits.resources.message
 const val GROUP_DISPLAY_ID = "AWS Toolkit"
 
 fun Exception.notifyError(title: String = "", project: Project? = null) =
-        notify(
-                Notification(
-                        GROUP_DISPLAY_ID,
-                        title,
-                        this.message ?: "${this::class.java.name}${this.stackTrace?.joinToString("\n", prefix = "\n")}",
-                        NotificationType.ERROR
-                ), project
-        )
+    notify(
+        Notification(
+            GROUP_DISPLAY_ID,
+            title,
+            this.message ?: "${this::class.java.name}${this.stackTrace?.joinToString("\n", prefix = "\n")}",
+            NotificationType.ERROR
+        ), project
+    )
```

Comment:
Minor: can we move these to the util file as well, so that all the `notify warn` can share the same behavior.